api/mapping/forms.py

```diff
@@ -3,6 +3,8 @@
 from django.contrib.auth.models import User
 from django.contrib.auth import password_validation
 from django.core.exceptions import ValidationError
+import csv
```

Comment:
Hi @vpnu, could you please organise the imports? On PyCharm we can use Code > Organise Imports. In the cli you could use isort.